Merging current develop to master

ritwiksen opened the pull request.

sasomarouthu committed:
close button in congrats modal popup

sasomarouthu committed:
issues Tech-3,4,6,7,9,23,24 hve been fixed

ritwiksen committed:
Merge pull request #20 from rajammaneesh/task/Tech-2

Issues Tech 3,4,6,7,9,23,24 have been fixed

ritwiksen committed:
Fix for Task Applications Load on Contributor screen

rajammaneesh committed:
Merge pull request #21 from rajammaneesh/task/Tech-28

Fix for Task Applications Load on Contributor screen

ritwiksen committed:
created new class active-side-nav

ritwiksen committed:
Added code updates

rajammaneesh committed:
Merge pull request #23 from rajammaneesh/task/service-line-filter-for-search

Added code updates

rajammaneesh committed:
Merge pull request #22 from rajammaneesh/task/Tech-20

created new class active-side-nav

ritwiksen committed:
Visibility of expand your skillset grid

sasomarouthu committed:
arranged search and service line dd in same line.

ritwiksen committed:
Merge pull request #25 from rajammaneesh/task/Enhancements

arranged search and service line dd in same line.

sasomarouthu committed:
Changed the style name to a meaning ful name

ritwiksen committed:
Modifications to filtering logic

sasomarouthu committed:
bug fixed from sheet nov 2017 defects from google docs

sasomarouthu committed:
Merge branch 'develop' into task/Enhancements

# Conflicts:
#	DCode.Web/Views/Contributor/_ContributorTaskApplication.cshtml

ritwiksen committed:
Merge pull request #26 from rajammaneesh/task/Enhancements

Changed the style name to a meaningful name

ritwiksen committed:
Merge with develop

ritwiksen committed:
Search to default from recommended search

ritwiksen committed:
Removed junk code

shrastogi committed:
Merge pull request #27 from rajammaneesh/task/service-line-filter-for-search

Task/service line filter for search

ritwiksen committed:
Fixes from Common Service and Identity Config

shrastogi committed:
Fixed the performance and Reliability issues.

shrastogi committed:
Merge pull request #28 from rajammaneesh/task/VS-profiler-fixes-ritwik

Fixes from Common Service and Identity Config

ritwiksen committed:
Merge pull request #29 from rajammaneesh/bugfix/CodeAnalysisFixes

Fixed the performance and Reliability issues.

ritwiksen committed:
Updated Tech X redirection link

ritwiksen committed:
Merge branch 'develop' of https://github.com/rajammaneesh/TestProject.git

ritwiksen committed:
Updated Logos

sasomarouthu committed:
added SOP in requestor task application

sasomarouthu committed:
Merge branch 'develop' into task/Enhancements

ritwiksen committed:
WIP

ritwiksen committed:
Configurable Tech X artifacts

ritwiksen committed:
Boolean check

shrastogi committed:
Merge pull request #31 from rajammaneesh/task/configurable-tech-x-access

Configurable Tech X artifacts

ritwiksen committed:
Added code to generate bulk emails

ritwiksen committed:
Added code to fetch subscribed users

ritwiksen committed:
Updating the code for inserting notifications subscription status

ritwiksen committed:
Notifications value retention on load of Profile Controller page

ritwiksen committed:
Merge branch 'develop'

ritwiksen committed:
Script for notification subscriptions

ritwiksen committed:
Merge pull request #30 from rajammaneesh/task/Enhancements

 added SOP in requester task application

ritwiksen committed:
Added daily usage statistics to the schema

ritwiksen committed:
Add Daily Statistics Count

ritwiksen committed:
Repository code to update Daily Usage Statistics

ritwiksen committed:
Service Call to fetch daily usage counts

ritwiksen committed:
code to insert and fetch database records

rajammaneesh committed:
Merge pull request #32 from rajammaneesh/task/bulk-email-notification

Task/bulk email notification

shrastogi committed:
1. Added check to prevent user from entering his email id in manager's email text box
2. Corrected the notification text on permissions page for requestor

shrastogi committed:
bug fix.

shrastogi committed:
Fixed the defect where assigned notification email was not going to contributor's manager.

rajammaneesh committed:
Merge pull request #33 from rajammaneesh/bugfix/RecommendedFixes

1. Added check to prevent user from entering his email id in manager'…

ritwiksen committed:
Bulk Notification functionality integrated

ritwiksen committed:
Got the Console Application Running

Commit:
dbchanges

sasomarouthu committed:
added subscription checkbox

rajammaneesh committed:
Merge pull request #34 from rajammaneesh/task/Enhancements

added subscription checkbox

sasomarouthu committed:
changed permissions to approvals

rajammaneesh committed:
Merge pull request #35 from rajammaneesh/task/Enhancements

changed permissions to approvals

Commit:
text change

sasomarouthu committed:
design issue fixes

rajammaneesh committed:
Merge pull request #36 from rajammaneesh/task/Enhancements

design issue fixes

Commit:
fix for con navigating to profile instead of Dashboard

Commit:
Merge branch 'develop' of https://github.com/rajammaneesh/TestProject.git

ritwiksen committed:
Notifications email console application added

ritwiksen committed:
removed Thread.Sleep

rajammaneesh committed:
Merge pull request #37 from rajammaneesh/task/bulk-email-notification

Task/bulk email notification

ritwiksen committed:
Merge with develop and added new table change

ritwiksen committed:
Updated Link and added additional binding

rajammaneesh committed:
Merge pull request #38 from rajammaneesh/task/daily-usage-analytics

Task/daily usage analytics

Commit:
sql file latest

ritwiksen committed:
Code fixes

rajammaneesh committed:
Merge pull request #39 from rajammaneesh/task/recommended-skill-filter-no-skillset

Code fixes

Commit:
Remove commented code and fix project manager name fix

Commit:
remove test code

ritwiksen committed:
Merge pull request #40 from rajammaneesh/ProjectManagerNameFix

Project manager name fix

Commit:
change the limit

Commit:
Merge branch 'develop' of https://github.com/rajammaneesh/TestProject.git

Commit:
fix close notification

sasomarouthu committed:
removed 2weeks validation and max hrs limit

sasomarouthu committed:
added validation for invalid skill entered

rajammaneesh committed:
Merge pull request #41 from rajammaneesh/task/IssuesFixes_Nov12017

removed 2weeks validation and max hrs limit

rajammaneesh committed:
Merge pull request #42 from rajammaneesh/task/IssuesFixes_Nov12017

added validation for invalid skill entered

shrastogi committed:
Fixed the issues found in smoke testing.

1. Removed MySeviceLine search filter.
2. Added RM group email in Assign notification

Commit:
skills getting qiped from db - issue fixed

rajammaneesh committed:
Merge pull request #43 from rajammaneesh/bugfix/SmokeTestBugfixes

Fixed the issues found in smoke testing.

shrastogi committed:
Merge pull request #44 from rajammaneesh/task/Golive_Fixes

skills getting qiped from db - issue fixed

sasomarouthu committed:
added contact page and the required code

rajammaneesh committed:
Merge pull request #45 from rajammaneesh/task/IssuesFixes_Nov12017

added contact page and the required code

Commit:
fix task status -active tasks should be shown

Commit:
Merge branch 'develop' of https://github.com/rajammaneesh/TestProject.git

Commit:
hot fixes for golive

ritwiksen committed:
Added Logs

ritwiksen committed:
Merge branch 'develop' of https://github.com/rajammaneesh/TestProject.git

ritwiksen committed:
shifted div and added ng-cloak

Commit:
Task History records fix

Commit:
Merge branch 'develop' of https://github.com/rajammaneesh/TestProject.git

rajammaneesh committed:
Merge pull request #47 from rajammaneesh/task/shift-subscriptions-to-editable-panel

shifted div and added ng-cloak

Commit:
userrepo cleanup + visit count changes

ritwiksen committed:
Fixes for MD login

rajammaneesh committed:
Merge pull request #49 from rajammaneesh/task/fixes-for-managing-director-login

Fixes for MD login

ritwiksen committed:
Merge pull request #48 from rajammaneesh/task/UserRepo_Cleanup

userrepo cleanup + visit count changes

Commit:
contact us page ui fixes

Commit:
1. Changed the skill search from startswith to contains
2. Increased the minlength from 1 to 3

ritwiksen committed:
Merge pull request #50 from rajammaneesh/task/Skills_Contains

1. Changed the skill search from startswith to contains

ritwiksen committed:
task/data-viewer-page

ritwiksen committed:
Removed additional line

ritwiksen committed:
Added entry for delete query

shivam-r committed:
Merge pull request #51 from shivam-r/task/query-viewer-page

task/data-viewer-page

ritwiksen committed:
Modifying regex for WBS code

shivam-r committed:
Merge pull request #52 from shivam-r/task/adding-new-wbs-format

Modifying regex for WBS code

ritwiksen committed:
Added new AD property mapping

ritwiksen committed:
Merge pull request #53 from shivam-r/task/fixing-userid

Added new AD property mapping

ritwiksen committed:
Adding task type column

ritwiksen committed:
Removed elmah error for test

ritwiksen committed:
Adding back elmah_error

shivam-r committed:
Merge pull request #54 from shivam-r/task/adding-task-type-column

Task/adding task type column

ritwiksen committed:
Adding missing daily usage table script

shivam-r committed:
Merge pull request #55 from shivam-r/task/adding-task-type-column

Adding missing daily usage table script

ritwiksen committed:
Added Firm Initiative Notification Operation

ritwiksen committed:
Fetching firm initiatives for notifications

ritwiksen committed:
Added factories for various implementations

ritwiksen committed:
Added remaining code for Firm Initiative Notification

ritwiksen committed:
Debugged Firm Initiative tasks

ritwiksen committed:
Removed commented code

ritwiksen committed:
Changed default option for loading tasks

shivam-r committed:
Merge pull request #57 from shivam-r/task/default-view-task-to-all-service-line

Changed default option for loading tasks

Commit:
Enhancements:: Implemented Close Task functionality for requestor; Changed the task status to close for past due date tasks;

ritwiksen committed:
Fixed regex

Commit:
Bug fix

Commit:
Merge branch 'task/Enhancements' of https://github.com/shivam-r/TestProject.git

shivam-r committed:
Merge pull request #56 from shivam-r/task/firm-initiative-notifications

Task/firm initiative notifications

shivam-r committed:
Added tasktype dropdown on create new task page.

shivam-r committed:
Merge pull request #59 from shivam-r/feature/CreateTaskTypeDropDown

Added tasktype dropdown on create new task page.

ritwiksen committed:
Added FI drop down for my tasks in Requestor Login

ritwiksen committed:
Corrected task type id

ritwiksen committed:
Added components for task history

ritwiksen committed:
Added filter for requestor for task status

ritwiksen committed:
Task Type filter added

shivam-r committed:
Merge pull request #60 from shivam-r/task/cs-fi-lov

Task/cs fi lov

ritwiksen committed:
Added code to implement create task

shivam-r committed:
Merge pull request #58 from shivam-r/task/Enhancements

Task/enhancements

shivam-r committed:
Added task type filter on contributor dashboard.

shivam-r committed:
Apply Task workflow for FI.

ritwiksen committed:
Merge with develop

shivam-r committed:
Merge pull request #61 from shivam-r/task/firm-initiative-in-new-task

Added code to implement create task

shivam-r committed:
Merge pull request #62 from shivam-r/feature/ContributorTaskTypeFilter

Added task type filter on contributor dashboard.

shivam-r committed:
Merge pull request #63 from shivam-r/feature/OverrideApprovalProcessForFITask

Feature/override approval process for fi task

shivam-r committed:
Fixed build issue

ritwiksen committed:
Removed task type id hard coding

shivam-r committed:
Merge pull request #64 from shivam-r/task/Corrected-hardcoded-task-type-id

Removed task type id hard coding

Commit:
Removed days ago text from duration message

shivam-r committed:
Merge pull request #66 from shivam-r/task/RequestorTaskStatusMessageModification

Removed days ago text from duration message

sasomarouthu committed:
design changes in requestor and contributor in dashboard and task application pages

shivam-r committed:
Merge pull request #67 from shivam-r/Design_Changes

design changes in requestor and contributor in dashboard and task application pages

ritwiksen committed:
Fixes to notification job

shivam-r committed:
Merge pull request #68 from shivam-r/task/fixes-to-notification-job

Fixes to notification job

shivam-r committed:
enabled Requestor role for level 3 and level 4 designations for RM and Senior Cons/Specialists

Commit:
Fix for exception in case of no new skills

shivam-r committed:
Fix for add time issue on contributor track status page.

shivam-r committed:
Merge pull request #70 from shivam-r/task/FirmInitiativeFix

Fix for exception in case of no new skills

Commit:
Bugfix:: Fix for moving the task type field in first row and WBS code field in second row on "Create new task" page for Requestor

Commit:
RequestorInEmailCCFix:: Implementation for adding requestor in email cc when task approved/rejected by manager

Commit:
Implementation for changing field name depending on task type i.e. FI/Project

shivam-r committed:
Merge pull request #72 from shivam-r/task/Bugfixes

Bugfix:: Fix for moving the task type field in first row and WBS code…

shivam-r committed:
Merge pull request #73 from shivam-r/task/RequestorInEmailCCFix

RequestorInEmailCCFix:: Implementation for adding requestor in email …

shivam-r committed:
Merge pull request #74 from shivam-r/task/FixForFieldName

Implementation for changing field name depending on task type i.e. FI…

priyankalekkala committed:
Skill drop down defaulted in case of FI on new task page

shivam-r committed:
Merge pull request #75 from shivam-r/bugfix/SkillDDLFix

Skill drop down defaulted in case of FI on new task page

Commit:
Fix for hiding task with due date past 15 days

ritwiksen committed:
Addind DL's for Consulting

ritwiksen committed:
Added test flow check

ritwiksen committed:
PR review changes

shivam-r committed:
Merge pull request #77 from shivam-r/task/consulting-users-for-firm-initiative-notification

Task/consulting users for firm initiative notification

Commit:
Incorporated review comments

shivam-r committed:
Merge pull request #76 from shivam-r/task/RemoveTasksOlderThan15Days

Fix for hiding task with due date past 15 days

Commit:
Update for Comments

shivam-r committed:
Merge pull request #78 from shivam-r/task/RemoveTasksOlderThan15Days

Update for merging query to reduce db calls

ritwiksen committed:
Html fix

shivam-r committed:
Merge pull request #79 from shivam-r/task/fixes-to-notification-email

Html fix

ritwiksen committed:
Client service tasks html generator

shivam-r committed:
Merge pull request #80 from shivam-r/task/fix-for-client-service-tasks-html

Client service tasks html generator

ritwiksen committed:
task/fix-for-default-skill-for-new-fi-task

ritwiksen committed:
Revert "task/fix-for-default-skill-for-new-fi-task"

This reverts commit 4bc5d8266d08128f890cca98fdd20e6955aa8a59.

ritwiksen committed:
Code updates for empty FI skill

shivam-r committed:
Merge pull request #81 from shivam-r/task/default-fi

Task/default fi

priyankalekkala committed:
Pushing UI fixes for FI

ritwiksen committed:
Code for sending FI assignment email

ritwiksen committed:
Fix for Assigned task

ritwiksen committed:
Sending new firm initiative notification to the floor

shivam-r committed:
Merge pull request #82 from shivam-r/Fixes-UI

Pushing UI fixes for FI

shivam-r committed:
Merge pull request #83 from shivam-r/task/fixes-from-10-5-2018-demo

Task/fixes from 10 5 2018 demo

ritwiksen committed:
Added additional content for New Fi emaiiler

shivam-r committed:
Merge pull request #84 from shivam-r/task/changing-fi-mail-content

Added additional content for New Fi emaiiler

priyankalekkala committed:
updated the notification email formatting and added new data

priyankalekkala committed:
Updated the notification email subject

shivam-r committed:
Merge pull request #85 from shivam-r/MailFix

Mail fix

ritwiksen committed:
Added code for implementation of modified firm initiatives notifications

shivam-r committed:
Merge pull request #86 from shivam-r/task/modified-firm-initiative-notifications-implementation

Added code for implementation of modified firm initiatives notifications

shivam-r reviewed (changes requested):
Config files needs to be deleted

ritwiksen committed:
Database updates for OpShift

shivam-r committed:
Merge pull request #88 from shivam-r/task/OpShift-db-schema-updates

Database updates for OpShift

ritwiksen committed:
Updated Db Schema to store Offering Code, Value and RM and Offering DL email Id's in database. Also Portfolio code and value will be stored in database.

shivam-r committed:
Merge pull request #89 from shivam-r/task/OpShift-db-schema-updates

Updated Db Schema to store Offering Code, Value and RM and Offering D…

Commit:
Service line to offering changes

Commit:
Included offering and portfolio in dbcontext class

priyankalekkala committed:
New Logo changes

priyankalekkala committed:
Get Offerings from database

shivam-r committed:
Merge pull request #91 from shivam-r/LogoFix

New Logo changes

priyankalekkala committed:
Offering Model Factory changes

shivam-r committed:
Merge pull request #92 from shivam-r/Offerings

Offerings

Commit:
Resolved conflicts

Commit:
Review comments changes

shivam-r committed:
Merge pull request #93 from shivam-r/task/ServiceLine-Namechange

Task/service line namechange

Commit:
code updates

shivam-r committed:
Implementation to replace SL with offering on various screens.

Implementation for sending EMail to RM group on application and OfferingDL on FI creation

shivam-r committed:
Merge pull request #95 from shivam-r/feature/PR-12072018

Offering Portfolio feature

Commit:
Debugged scenarios for bad email id's

Commit:
Split email list to string

Commit:
Merge with develop

shivam-r committed:
Merge pull request #96 from shivam-r/task/collective-client-service-task-notification

Task/collective client service task notification

Commit:
Changed TechX to TX across mails

Commit:
Offerings text added to all notifications

Commit:
Added code for default consulting mailbozes

Commit:
Send dummy mails to consulting if practice emails are unavailable

Commit:
Fixed issue for notification emailer not getting sent

Commit:
Resized image

Commit:
Resized email images

shivam-r committed:
Merge pull request #97 from shivam-r/task/uat-fixes-tx

Task/uat fixes tx

shivam-r committed:
Fixed the issues with display of offering name.

shivam-r committed:
Merge pull request #98 from shivam-r/bugfix/PR-17072018

Fixed the issues with display of offering name.

Commit:
Class modifications

shivam-r committed:
Merge pull request #99 from shivam-r/task/logo-size-fix

Task/logo size fix

Commit:
Added bold formatting for replaced values in mail body

shivam-r committed:
Merge pull request #100 from shivam-r/task/email-formatting

Added bold formatting for replaced values in mail body

Commit:
Updating TX to TechX text

Commit:
Uncommenting code

shivam-r committed:
Merge pull request #101 from shivam-r/task/email-formatting

Updating TX to TechX text

priyankalekkala committed:
Revereted the logo changes

priyankalekkala committed:
Revert PR changes

Commit:
Added code to check unauthorized access

Commit:
Corrected condition

shivam-r committed:
Uncommented required code

shivam-r committed:
Merge pull request #102 from shivam-r/LogoRevertChanges

Revereted the logo changes

Commit:
Configured code from database

shivam-r committed:
Merge pull request #103 from shivam-r/task/add-unauthorize-check

Task/add unauthorize check

Commit:
Getting default emails from database

shivam-r committed:
Merge pull request #106 from shivam-r/task/get-default-emailids-from-database

Getting default emails from database

shivam-r committed:
Upgraded .NET Framework
Upgraded Entity Framefork
Upgraded .NET MYSQL Connector
Implemented Industry Initiative feature

shivam-r committed:
Merge pull request #107 from shivam-r/feature/IndustryInititative

Industry Initiative

Commit:
Adding methods to troubleshoot

shivam-r committed:
Merge pull request #108 from shivam-r/task/Added-logs-for-cs-collective-task

Adding methods to troubleshoot
